core/templates/pages/collection-editor-page/services/collection-editor-state.service.ts

```diff
@@ -20,26 +20,24 @@
 
 require('domain/collection/CollectionObjectFactory.ts');
 require('domain/collection/collection-rights-backend-api.service.ts');
-require('domain/collection/CollectionRightsObjectFactory.ts');
 require('domain/collection/editable-collection-backend-api.service.ts');
 require('domain/editor/undo_redo/undo-redo.service.ts');
 require('services/alerts.service.ts');
 
 require('pages/collection-editor-page/collection-editor-page.constants.ajs.ts');
 
 import { EventEmitter } from '@angular/core';
-
+import { CollectionRights } from 'domain/collection/CollectionRightsObjectFactory';
```

Comment:
new line

Reply:
Please address this comment.

Reply:
check the last commit I made the cnanges

Reply:
There is still no empty line.

Reply:
done!